### PR TITLE
Safe to using rake 11.0.1 now.

### PR DESCRIPTION
**Pull request by Eric-Guo**

### CODE REVIEW

**Comment by remear** [comment]
Awesome! (sans the jruby failures)

@rails-api/ams should we go back to not specifying a dev dependency for rack, or add a sensible range?

File: active_model_serializers.gemspec [hunk 1]
```diff
@@ -56,6 +56,5 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'minitest-reporters'
   spec.add_development_dependency 'grape', ['>= 0.13', '< 1.0']
   spec.add_development_dependency 'json_schema'
-  # https://stackoverflow.com/questions/35893584/nomethoderror-undefined-method-last-comment-after-upgrading-to-rake-11/35893625#35893625
-  spec.add_development_dependency 'rake', '< 11.0'
+  spec.add_development_dependency 'rake'
```
Comment (bf4) [inline active_model_serializers.gemspec]: `['>= 10.0', '< 12.0']` maybe

**Comment by padi** [comment]
:+1: since 11.0.0 is yanked.

**Comment by NullVoxPopuli** [comment]
woo, I'll re-run the build

**Comment by NullVoxPopuli** [comment]
oh... Jruby.....

**Comment by NullVoxPopuli** [comment]
can you rebase this with master? I think the Jruby issues are fixed.

**Pull request merged**